### PR TITLE
Dev feature update descriptions

### PR DESCRIPTION
**Pull request by mtammineni**
Update descriptions for Undelivered Uploads.
Update UpsertReport descriptors.
Update README file.

### CODE REVIEW

File: pstatus-graphql-ktor/README.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
 # Overview
-This project is the processing status report sink.  It listens for messages on an Azure Service bus, validates the messages, and if validated persists them to CosmosDB.
+This project is the processing status service. The pstatus-graph-ktor service provides the following features:
```
Comment (manu-govind) [inline pstatus-graphql-ktor/README.md:2]: " graphQL service " missing in the first sentence?

File: pstatus-graphql-ktor/README.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+
+Ability to query for existing reports, uploads, upload status.
```
Comment (manu-govind) [inline pstatus-graphql-ktor/README.md:4]: need to mention about dead-letter reports as well

Comment (manu-govind) [reply pstatus-graphql-ktor/README.md:4]: need to mention about notifications as well

File: pstatus-graphql-ktor/README.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+### Security 
+SECURITY_ENABLED - Boolean value. Set to false for development purposes in the local development environments.
+
+### Notifications
+PSTATUS_NOTIFICATIONS_BASE_URL - The notifications service base url.
+
+### Workflows
+PSTATUS_WORKFLOW_NOTIFICATIONS_BASE_URL - The workflows service base url.
+
+
```
Comment (manu-govind) [inline pstatus-graphql-ktor/README.md:31]: Here isn't it better to give an overview of all the endpoints - at least what they do - maybe copy paste from the playground documentation?

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/services/ReportMutationService.kt [hunk 4]
```diff
@@ -55,7 +55,7 @@ class ReportMutationService : CosmosLoader() {
      * @throws ContentException If there is an error with the content format.
      */
     @Throws(BadRequestException::class, ContentException::class, Exception::class)
-    fun upsertReport(input: ReportInput, action: String): Report? {
+    fun upsertReport(action: String, input: ReportInput,): Report? {
```
Comment (manu-govind) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/services/ReportMutationService.kt:58]: Is there an extra unwanted "comma" after ReportInput?

**Review by manu-govind — CHANGES_REQUESTED** [review]

File: pstatus-graphql-ktor/README.md (hunk 1 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/README.md]: Corrected the description.

File: pstatus-graphql-ktor/README.md (hunk 2 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/README.md:5]: Added descriptions for Dead letter Reports and Notifications.

File: pstatus-graphql-ktor/README.md (hunk 3 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/README.md:103]: Added the endpoints and the respective description available for each of the endpoints from the GraphQL Descriptions.

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/services/ReportMutationService.kt (hunk 4 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/services/ReportMutationService.kt]: Removed the extra comma.

**Pull request merged**